components/toggle/Toggle.js

```diff
@@ -1,28 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { t } from 'ttag';
+import { c } from 'ttag';
 
 import Input from '../input/Input';
-import Label from '../label/Label';
+import Icon from '../icon/Icon';
 
-const Toggle = ({ id }) => {
+const label = (type, key) => {
+    const I18N = {
+        on: c('Toggle button').t`On`,
+        off: c('Toggle button').t`Off`
+    };
+
+    if (type === 'icon') {
+        return (
+            <span className="pm-toggle-label-text">
+                <Icon name={key} alt={I18N[key]} className="pm-toggle-label-img" />
+            </span>
+        );
+    }
+
+    if (key === 'on') {
```

Comment:
If the design is using `ON` / `OFF` inside a `<Toggle>`, it's a mistake, we always need to use icon.